src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -1401,7 +1401,8 @@ class NonNullTypesAttribute : Attribute
             comp.VerifyDiagnostics();
         }
 
-        [Fact(Skip = "PROTOTYPE(NullableReferenceTypes): stack overflow")]
+        [WorkItem(29594, "https://github.com/dotnet/roslyn/issues/29594")]
+        [Fact(Skip = "Stack overflow")]
```

Comment:
>Stack overflow [](start = 22, length = 14)

Here and in other similar places, should we use the link to the issue as the Skip reason?

Reply:
Having a short description avoids the need to look up the bug. Will leave as is, with link in the `WorkItem` and description in `Skip = "..."`.

---
In reply to: [214486306](https://github.com/dotnet/roslyn/pull/29607#discussion_r214486306) [](ancestors = 214486306)